charts/github-bot-safe-settings/templates/app-env.yaml

```diff
@@ -0,0 +1,22 @@
+{{- $fullName := include "safe-settings.fullname" . -}}
+{{- if .Values.appEnv.create }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "safe-settings.fullname" . }}-app-env
+type: Opaque
+data:
+  APP_ID: {{ required "A valid appEnv.APP_ID is required!" .Values.appEnv.APP_ID | toString | b64enc }}
```

Comment:
Please check out how @dmitry-mightydevops added `required` logic to `terraform-pod` helm chart. Let's do it in the same way to not repeat the same `required` stuff all the time where variable is used and simplify chart manifests. He did it in this way:

1. He defined `terraform-pod.validateValues` in `_helpers.tpl` and added `required` logic checks there - https://github.com/saritasa-nest/saritasa-devops-helm-charts/blob/main/charts/terraform-pod/templates/_helpers.tpl#L309
2. He used this tpl in NOTES.txt and that's all (so he don't have `required ...` stuff in yaml manifests and his code looks much simpler because of that) - https://github.com/saritasa-nest/saritasa-devops-helm-charts/blob/main/charts/terraform-pod/templates/NOTES.txt#L1